dumpling/export/writer_serial_test.go

```diff
@@ -52,7 +52,8 @@ func TestWriteInsert(t *testing.T) {
 	bf := storage.NewBufferWriter()
 
 	conf := configForWriteSQL(cfg, UnspecifiedSize, UnspecifiedSize)
-	n, err := WriteInsert(tcontext.Background(), conf, tableIR, tableIR, bf, defaultMetrics)
+	m := newMetrics(conf.PromFactory, conf.Labels)
```

Comment:
seems we can remove cfg.Labels now because DM should use constLabels Factory, or we can do it later

Reply:
It's useful for someone who just wants to use the original prometheus factory.